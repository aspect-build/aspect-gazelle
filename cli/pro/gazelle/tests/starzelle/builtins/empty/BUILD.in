load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")

copy_to_bin(
    name = "ctb",
)

sh_library(
    name = "shells",
)

sh_binary(
    name = "shells-bin",
)

sh_test(
    name = "shells-test",
)

filegroup(
    name = "all-files",
)
