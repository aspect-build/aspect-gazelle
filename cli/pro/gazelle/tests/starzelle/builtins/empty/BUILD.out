sh_test(
    name = "shells-test",
)
